ui/scripts/docs.js

```diff
@@ -1008,7 +1008,7 @@ cloudStack.docs = {
     },
     // Add volume
     helpVolumeName: {
-        desc: 'Give the volume a unique name so you can find it later.',
+        desc: 'Give a unique volume name. If it is not provided, a name will be generated randomly.',
```

Comment:
Yes that would be better to put this sentences into dictionary and messages.properties (key should start with "message.tooltip.xxxx") 
(even if they are a lot of 'desc' strings in javascript files which need to be localized. Probably a work for me ;-))